recipes/nextcloud-talk/webview.js

```diff
@@ -5,21 +5,24 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 module.exports = Ferdium => {
   const getMessages = () => {
     let direct = 0;
+
     const notificationWrapper = document.querySelector(
       '.notifications .notification-wrapper',
     );
 
     if (notificationWrapper) {
-      direct = notificationWrapper.querySelectorAll(
+      const directSelector = notificationWrapper.querySelectorAll(
         '.notification[object_type="chat"], .notification[object_type="room"',
```

Comment:
Please wait until PR #64 is merged. This line needs to be corrected accordingly.

Reply:
Merge solved